packages/api-client/src/cumulusApiClient.ts

```diff
@@ -0,0 +1,53 @@
+import pRetry from 'p-retry';
+import Logger from '@cumulus/logger';
+import { lambda } from '@cumulus/aws-client/services';
+import { CumulusApiClientError } from './CumulusApiClientError';
+import * as types from './types';
+
+const logger = new Logger({ sender: '@api-client/cumulusApiClient' });
+
+/**
+ * Invoke the Cumulus private API lambda (using pRetry)
+ *
+ * @param {Object} params
```

Comment:
Not that it needs done in *this* pr, but noted the jsdocs were updated here including types - can we just get rid of them given we're explicitly typing them?

Reply:
As @markdboyd  just reminded me, we have an issue with generated docs, so perhaps never mind.   🤔